MediaGallerySynchronization/Model/CreateAssetFromFile.php

```diff
@@ -67,9 +76,11 @@ public function execute(\SplFileInfo $file)
         $path = $file->getPath() . '/' . $file->getFileName();
 
         [$width, $height] = getimagesize($path);
+        
 
         return $this->assetFactory->create(
             [
+                'id' => $this->getAssetId($path),
                 'path' => $this->getPath($path),
```

Comment:
```suggestion
        $path = $file->getPath() . '/' . $file->getFileName();
        $relativePath = $this->getRelativePath($path);
        [$width, $height] = getimagesize($path);

        return $this->assetFactory->create(
            [
                'id' => $this->getAssetId($relativePath),
                'path' => $this->getPath($path),
```

Reply:
If we get $relativePath once at line 84 we would avoid calling it several times for the same result.